src/calculateRentalCost.js

```diff
@@ -20,7 +20,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costFor1Day = 40;
+  const discountFor7Days = 50;
+  const discountFor3Days = 20;
+
+  switch (true) {
+    case days >= 7:
+      return days * costFor1Day - discountFor7Days;
```

Comment:
```suggestion
      return (days * costFor1Day) - discountFor7Days;
```
Не забывай расставлять скобки